### DISCUSSION TITLE
How to migrate database?

### DISCUSSION THREAD

**Discussion by ajl236** [requarks/wiki#2866] [Help / Questions] [answered]
We have WikiJS running against MS SQL server but want to migrate this content to PostgreSQL.  What's the best way to achieve this please ?

  Ubuntu 14.0, PostGres 9.3, WikiJS 2.5.170

**Comment by NGPixel** [comment, marked as answer]
At the moment, there's no easy way to migrate from one engine to another. You can however migrate content between 2 installations using either the Local File System or Git storage module.

**Reply by scienceasdf** [reply]
If I want to migrate users, is it okay to copy the entire user table from origin database to another database?

**Reply by Leoi-JR** [reply]
I have pushed my data to github. And I alse pulled these data from github to another pc, which has installed a new wikijs. How can I migrate these data to the new wikijs. Thanks!

**Comment by rockonedege** [comment]
Any updates on this feature?

I started with SQLite for the team. Now the user number grows and I'd like to migrate to a server-based database. How can I achieve it without losing any existing settings?

**Reply by NGPixel** [reply]
You'll be able to migrate when going from 2.x to 3.x. It's not possible to migrate db engines between 2.x installations at the moment.

**Reply by rockonedege** [reply]
That's great news. is there a release schedule?
